server/controllers/auth/signupHelpers/createParent.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  createUser, createParent, findUserByEmail, createStudent,
+} from '../../../queries';
+import { CustomError, UserTableInterface } from '../../../utils';
+
+const createParentAccount = async (user: UserTableInterface, children?: Array<string>) => {
+  const parentUser = await createUser(user);
+  const parentId = parentUser.getDataValue('id');
+
+  children?.forEach(async (child) => {
+    const doesChildStudent = await findUserByEmail(child);
+    const childId = doesChildStudent?.getDataValue('id');
+
+    if (!doesChildStudent) {
+      throw new CustomError(422, 'The email does not exist!');
+    }
+
+    await createStudent(childId, parentId);
+    await createParent(parentId);
+  });
+};
```

Comment:
Helpers not in the controller folder, move this function to the same file where the signup is. 
And it's a part of your controller it's not a helper.